### PR TITLE
Update case to match resource.

### PR DESCRIPTION
**Pull request by Jamstah**
https://github.com/openshift/api/blob/a99ffa1cac6709edf8f502b16890b16f9a557e00/console/v1/types_console_yaml_sample.go

Signed-off-by: James Hewitt <james.hewitt@uk.ibm.com>

**Description of the change:**

Update string to match the real world

**Motivation for the change:**

Kind is not recognised because the case does not match the resource.

**Reviewer Checklist**
- [ ] Implementation matches the proposed design, or proposal is updated to match implementation
- [ ] Sufficient unit test coverage 
- [ ] Sufficient end-to-end test coverage
- [ ] Docs updated or added to `/docs` 
- [ ] Commit messages sensible and descriptive

### CODE REVIEW

**Comment by Jamstah** [comment]
I switched this to change just the value and not the constant, although I expect the current constant isn't being used (as its wrong anyway), so it might be better to make the breaking API change instead.

**Comment by Jamstah** [comment]
Hah, changing the value is considered incompatible too, so I'm going to go ahead and say "we should make an incompatible change here".

File: pkg/lib/bundle/supported_resources.go [hunk 1]
```diff
@@ -16,7 +16,7 @@ const (
 	PodDisruptionBudgetKind   = "PodDisruptionBudget"
 	PriorityClassKind         = "PriorityClass"
 	VerticalPodAutoscalerKind = "VerticalPodAutoscaler"
-	ConsoleYamlSampleKind     = "ConsoleYamlSample"
+	ConsoleYAMLSampleKind     = "ConsoleYAMLSampleKind"
```
Comment (dinhxuanvu) [inline pkg/lib/bundle/supported_resources.go:19]: Is the `Kind` part expected on the value side?

**Comment by dinhxuanvu** [comment]
Can you please rebase this PR against latest master? We recently fixed a kind test on CI. Thanks.

File: pkg/lib/bundle/supported_resources.go (hunk 1 above)
Comment (Jamstah) [reply pkg/lib/bundle/supported_resources.go]: No! Thank you. Copy paste bug from the repeated tests. I've been upgrading my go to be able to run the e2e locally.

**Comment by Jamstah** [comment]
> Can you please rebase this PR against latest master? We recently fixed a kind test on CI. Thanks.

Done.

**Comment by Jamstah** [comment]
/assign @dinhxuanvu

Assigning because you already seem to be looking at it

**Comment by timflannagan** [comment]
Related: https://github.com/operator-framework/operator-registry/pull/757.

**Comment by Jamstah** [comment]
@timflannagan rebased, can I have the lgtm back?

**Review by dinhxuanvu — APPROVED** [review]
/approve

**Comment by Jamstah** [comment]
@timflannagan I had to rebase this so it lost your LGTM, can you please provide another?

**Pull request merged**